styles.css

```diff
@@ -0,0 +1,80 @@
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+.heading-section {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: space-around;
+  padding: 30px 20px;
+}
+
+.header {
+  display: flex;
+  width: 100%;
+  margin: 0;
+  justify-content: space-between;
+  color: white;
+  background-color: #3c3a39;
+  text-decoration: none;
+  padding: 20px;
+}
+
+body {
+  background-color: #1c1a19;
+  margin: 0;
+}
+
+.logo {
+  color: white;
+  text-decoration: none;
+}
+
+.heading {
+  color: #ff6b00;
+  font-size: 50px;
+  font-family: Crete Round, var(--font-family-common);
+  margin: 10% 0 0;
+}
+
+.describe {
+  font-family: Roboto, var(--font-family-common);
+  font-style: normal;
+  font-weight: 500;
+  font-size: 32px;
+  color: #fff;
+}
+
+.headline {
+  font-family: Poppins, var(--font-family-common);
+  font-size: 20px;
+  line-height: 32px;
+  color: #fff;
+}
```

Comment:
- [ ] As requested by the previous reviewer, kindly make sure your typography looks exactly like the provided design. To use these special fonts, you must first import them at the top of your CSS file from [google fonts](https://fonts.google.com/) in order to use it.

     _**Example of how to import the google font**_
     `@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');`

    _**and you can use it like this you css**_
   `font-family: 'Poppins', sans-serif;`

    - ### Your design
     ![image](https://user-images.githubusercontent.com/87197412/205020782-86fb6916-5f6e-4b84-8153-09c1427f2118.png)

   - ### Figma design
   
    ![image](https://user-images.githubusercontent.com/87197412/205020887-6b13da3d-22be-4ac8-9bed-fd0d864ee324.png)